scala/src/com/google/idea/blaze/scala/sync/BlazeScalaSyncPlugin.java

```diff
@@ -70,7 +70,7 @@ public Set<LanguageClass> getSupportedLanguagesInWorkspace(WorkspaceType workspa
    */
   private static final Pattern LIBRARY_VERSION = Pattern.compile("(?<=[:\\-])\\d+\\.\\d+\\.\\d+[^:\\s]*");
 
-  private static final Pattern RUNTIME_LIBRARY = Pattern.compile("((?:scala|dotty|scala3)-library).+");
+  private static final Pattern RUNTIME_LIBRARY = Pattern.compile("((?:scala|dotty|scala3|org_scala_lang_scala)(?:_|-)library).+");
```

Comment:
```suggestion
  private static final Pattern RUNTIME_LIBRARY = Pattern.compile("((?:scala-|dotty-|scala3-|org_scala_lang_scala_)library).+");
```

What do you think about this? The regex would just be more precise

Reply:
Works with me 👍